docat/docat/app.py

```diff
@@ -290,6 +361,7 @@ def tag(project: str, version: str, new_tag: str, response: Response):
         return ApiResponse(message=f"Version {version} not found")
 
     if create_symlink(version, destination):
+        update_version_index_for_project(DOCAT_UPLOAD_FOLDER, DOCAT_INDEX_PATH, project)
```

Comment:
No, it won't duplicate the index for that project, update_version_index_for_project drops any existing records and then stores all versions and tags it finds. I chose to recreate the version index entirely, because I was afraid of inconsistencies that might arise and it just makes it easier to overview if only the whole index can be created at once. 

I think it might still be useful to have the tags, just because if you can search for versions then why not for tags